operators/routernetes-operator/routernetes-operator.package.yaml

```diff
@@ -1,5 +1,5 @@
 channels:
```

Comment:
@loganmc10  Please remove this file. It is not needed for bundle format

Reply:
Ok sorry i'm still learning... Without that file, how does it know what the default channel is?